eng/docker-compose/start-local-config.ps1

```diff
@@ -55,7 +55,7 @@ else {
 
     Write-Output "Starting locally-built DMS config service"
 
-    docker compose $files --env-file $EnvironmentFile up $upArgs
+    docker compose $files --env-file $EnvironmentFile -p dms-local up $upArgs
```

Comment:
`-p` is for project name, and it is what the group of containers is called when viewing in Docker Desktop. Without this, the group is simply called "docker-compose" (the directory name), which is confusing if there are other repositories with a `docker-compose` directory.

Reply:
This one is the deploying local config service and its dependencies, can we name it cms-local?

Reply:
done.

And I realized that i needed to test start and stop both for the DMS and CS powershell scripts. Running those made me realize that I had a problem that needed to be solved. Done.